tapestry.js

```diff
@@ -2026,6 +2164,37 @@ function exitViewMode() {
     startForce();
 }
 
+function generateTextNodeHTML(title, str) {
+    var lightboxContent = document.createElement("div");
+
+    var titleSection = document.createElement("div");
+    titleSection.setAttribute("id", "text-light-box-title");
+
+    var titleText = document.createElement("p");
+    titleText.appendChild(document.createTextNode(title));
+    titleSection.append(titleText);
+    lightboxContent.append(titleSection);
+
+    if (str) {
+        var paragraphSection = document.createElement("div");
+        paragraphSection.setAttribute("id", "text-light-box-paragraph");
+        paragraphArray = str.split("\n");
```

Comment:
Can you please change this to `\n\n`? And then in the body of the iteration, replace any single instances of `\n` with `<br>`. That way, people can still add single line breaks.

Reply:
Modified so that user can add line breaks